src/Features/Traits/Cookies.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Behat\Gherkin\Node\TableNode;
+use Illuminate\Contracts\Cookie\QueueingFactory as CookieFactory;
+use Illuminate\Http\Request;
+use PHPUnit\Framework\Assert;
+
+trait Cookies
+{
+    use Application;
+
+    /**
+     * @When I set cookie :name with value :value
+     * @When I create a cookie named :name with value :value
+     * @When I add a cookie :name with value :value
+     */
+    public function setCookie(string $name, string $value): void
+    {
+        $cookieFactory = $this->getContainer()->make(CookieFactory::class);
+        $cookieFactory->queue($name, $value);
+    }
+
+    /**
+     * @Then cookie :name should have value :value
+     * @Then the cookie :name should contain value :value
+     * @Then the cookie named :name should have the value :value
+     */
+    public function assertCookieValue(string $name, string $value): void
+    {
+        $request = $this->getContainer()->make(Request::class);
+        $cookieValue = $request->cookie($name);
```

Comment:
```suggestion
        $cookieValue = $this->request->cookie($name);
```
Maybe we should check existing request from HttpRequest for cookies?